.github/workflows/lint.yml

```
@@ -0,0 +1,33 @@
+name: Lint
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  markdownlint:
+    name: Check markdown for style issues
+    runs-on: ubuntu-latest
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      - uses: actions/checkout@v2
+      - name: Cache
+        uses: actions/cache@v2.1.4
+        with:
+          path: ~/.npm
+          key: ${{ runner.os }}-node-${{ hashFiles('**/package.json') }}
+          restore-keys: |
+            ${{ runner.os }}-node-
+      - uses: actions/setup-node@v2
+        with:
+          node-version: 12
+
+      - uses: xt0rted/markdownlint-problem-matcher@v1
+      - run: npx markdownlint-cli '**/*.md'
```

actually, now that we have a packkage.json we might as well move this here to make local dev easier
```suggestion
      - run: npm run markdownlint
```